tests/acceptance/pageObjects/favoritesPage.js

```diff
@@ -13,6 +13,16 @@ module.exports = {
       return this.navigate(this.url()).waitForElementPresent(
         this.page.FilesPageElement.filesList().elements.anyAfterLoading
       )
+    },
+    batchActionButtonIsNotDisplayed: function () {
+      return this.waitForElementNotPresent('@batchActionButton')
+    }
```

Comment:
this is generally not a good practice. In the page object function, always prefer getting/setting the status of page elements rather than just verifying the result. All assertions or verification should be in the context files not in the page objects.

for this function, you can refactor it just like `isButtonVisible` function from `shareWithMe.js` PageObject.
```suggestion
    getBactActionButtonPresentStatus: function () {
      // find the present status
      return presentStatus
    }
```